docs/docs/installation/architecture.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Architecture
+hide_title: true
+sidebar_position: 1
+version: 1
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Architecture
+
+This page is meant to give new administrators an understanding of Superset's components.
+
+## Components
+
+A Superset installation is made up of these components:
+1. The Superset application itself
+2. A metadata database to store Superset's data about users, charts, dashboards, etc.
+3. A caching layer (optional, but necessary for some features)
```

Comment:
this is called out twice in "components" and "optional component", i think it's a great idea to seperate the section, but let's have each component on just one side